python/pipeline/stack.py

```diff
@@ -977,6 +978,129 @@ def make(self, key):
         self.insert1({**key, 'resized': resized, 'lcned': lcned, 'sharpened': sharpened})
 
 
+@schema
```

Comment:
we should put this one in pipeline/shared.py, that's where alll our other method  lookups are defined. Probably also match the convention of the other methods there. They are all kind of like thios:
```python
@schema
class StackSegmMethod(dj.Lookup):
    definition = """ # methods for 3-d stack segmentations
    stacksegm_method         : tinyint
    ---
    name                        : varchar(16)
    details                     : varchar(255)
    language                    : enum('matlab', 'python')  # implementation language
    """
    contents = [
        [1, '3dconv', '3-d convolutional network plus watershed', 'python'],
        [2, '3dconv-ensemble', 'an ensemble of 3-d convolutional networks plus watershed', 'python']
    ]
```
Just to keep stuff consistent